Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- main.py	(revision )
+++ main.py	(revision )
@@ -14,7 +14,7 @@
     elif option == "4":
         find_multiple_category_object(state)
     elif option == "5":
-        advance_search(state)
+        advanced_search(state)
     elif option == "0":
         exit()
     else:
